common/types/mapper/proto/admin.go

```diff
@@ -1192,10 +1189,8 @@ func FromAdminUpdateDomainIsolationGroupsRequest(t *types.UpdateDomainIsolationG
 	if t == nil {
 		return nil
 	}
-	if t.IsolationGroups == nil {
-		return &adminv1.UpdateDomainIsolationGroupsRequest{}
-	}
 	return &adminv1.UpdateDomainIsolationGroupsRequest{
+		Domain:          t.Domain,
```

Comment:
Could you change the title to reflect this bug fix?

Reply:
Done